src/migrations/20191112101055-create-user.js

```diff
@@ -7,28 +7,81 @@ module.exports = {
       type: Sequelize.INTEGER
     },
     firstName: {
-      type: Sequelize.STRING
+      allowNull: false,
+      type: Sequelize.STRING,
     },
     lastName: {
-      type: Sequelize.STRING
-    },
-    email: {
-      type: Sequelize.STRING
+      allowNull: false,
+      type: Sequelize.STRING,
     },
     password: {
-      type: Sequelize.STRING
+      allowNull: false,
+      type: Sequelize.STRING,
+      defaultValue: 'password',
     },
     isVerified: {
+      allowNull: false,
       type: Sequelize.BOOLEAN,
-      defaultValue: false
+      defaultValue: false,
+    },
+    lastLogin: {
+      allowNull: true,
+      type: Sequelize.DATE
+    },
+    email: {
+      allowNull: false,
+      type: Sequelize.STRING,
+      unique: true,
+      validate: {
+        isEmail: true,
+      },
+    },
+    birthDate: {
+      allowNull: true,
+      type: Sequelize.DATE,
+    },
+    residenceAddress: {
+      allowNull: true,
+      type: Sequelize.STRING,
+    },
+    lineManager: {
+      allowNull: true,
+      type: Sequelize.STRING,
+    },
```

Comment:
Great work @gildniy, but I think this would have a type of integer and be a foreign key linking users and their manager

Reply:
I made this thinking a case where the line manager could be a non-registered user.

Reply:
I think that all the manager will be registered users on the platform

Reply:
Ok you're right, let me implement it that way, thanks for the suggestion.